### ISSUE TITLE
Webcam doesn't initialize in Teamtalk 5.5

### ISSUE THREAD

**Issue by pushpi004** [BearWare/TeamTalk5#584]
Hi all. I am facing issues with regards to the webcam of my laptab. While using Teamtalk 4.6, the webcam of my laptab works. But with the latest version (5.5), the webcam is not probably detected by Teamtalk. The preferences section doesn't list the webcam settings as earlier.
I am working on an iball notebook (k series) with windows 10 OS. The hardware configuration is decent. Rest of the features of TeamTalk works properly.

Please help.

Regards

**Comment by bear101** [comment]
Do you get any resolutions listed when you toggle between RGB32, I420 and YUY2?

**Comment by pushpi004** [comment]
Yes.. I did.. checked between all three options.. the webcam isn't listed in any of the options. Somehow teamtalk is not able to detect it. The webcam is accessible from other apps.
